dts/bindings/serial/zephyr,cdc-acm-uart.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: USB CDC ACM UART
```

Comment:
TBH, I do not want to do it now because there is no need for it. We do not have support for multiple controllers. And when the time comes, the CDC ACM classe will have to be re-implemented.

Reply:
> TBH, I do not want to do it now because there is no need for it.

I think if you added this, plus the BUILD_ASSERT I described [below](https://github.com/zephyrproject-rtos/zephyr/pull/37085#discussion_r683861679), you would have got build errors for the invalid use of aliases. So I disagree there is no need for it -- it can be used to catch programmer errors in the DTS files.